service/dynamodb/expression/expression.go

```diff
@@ -207,18 +241,23 @@ func (b Builder) WithKeyCondition(keyConditionBuilder KeyConditionBuilder) Build
 	return b
 }
 
-// WithFilter method will add the argument ConditionBuilder as a treeBuilder to
-// the argument Builder. If the argument Builder already has a
-// ConditionBuilder under the key filter, WithFilter() will overwrite the
-// existing ConditionBuilder. Users will able to add other treeBuilders to the
-// Builder or call Build() to build a Expression struct.
+// WithFilter method adds the argument ConditionBuilder as a Filter Expression
+// to the argument Builder. If the argument Builder already has a
+// ConditionBuilder representing a Filter Expression, WithFilter()
+// overwrites the existing ConditionBuilder. Users are able to add other
```

Comment:
target `you`, instead of `Users`.  Or consider reword to more declarative such as, `Additional expressions can be added to the Builder via the `With___` methods. To build an Expression, call Build.`